src/js/view/controls/components/chapters.mixin.js

```diff
@@ -33,8 +33,8 @@ const ChaptersMixin = {
     chaptersLoaded: function (evt) {
         const data = srt(evt.responseText);
         if (Array.isArray(data)) {
-            data.forEach((obj) => this.addCue(obj));
-            this.drawCues();
+            // Add chapter cues directly to model which will trigger addCue()
+            this._model.set('cues', data);
```

Comment:
This should append the new cues, or call a new `addCues` method which does the appending. Since you have direct access to the model, you could slice model cues, append and set, and then depend on the `change:cues` event to handle the update in the time slider chapters mixin.